File: app/src/main/java/org/pyload/android/client/module/AllTrustManager.java
package org.pyload.android.client.module;

import java.security.cert.X509Certificate;

import javax.net.ssl.X509TrustManager;

public class AllTrustManager implements X509TrustManager {


    public void checkClientTrusted(X509Certificate[] arg0, String arg1) {
    }


    public void checkServerTrusted(X509Certificate[] arg0, String arg1) {
    }


    public X509Certificate[] getAcceptedIssuers() {
        return null;
    }

}

File: app/src/main/java/org/pyload/android/client/module/AllTrustManager.kt
package org.pyload.android.client.module

import java.security.cert.X509Certificate

import javax.net.ssl.X509TrustManager

class AllTrustManager : X509TrustManager {

    override fun checkClientTrusted(chain: Array<X509Certificate>?, authType: String?) {}

    override fun checkServerTrusted(chain: Array<X509Certificate>?, authType: String?) {}

    override fun getAcceptedIssuers(): Array<X509Certificate>? {
        return null
    }
}